addons/binding/org.openhab.binding.freemobilesms/src/main/java/org/openhab/binding/freemobilesms/internal/FreeMobileSmsConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.freemobilesms.internal;
+
+/**
+ * The {@link FreeMobileSmsConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Guilhem Bonnefille <guilhem.bonnefille@gmail.com> - Initial contribution
```

Comment:
Please remove the e-mail address everywhere. Bots and outdated addresses have already caused problems.

Reply:
Removed